rune/rune
# feat: added cli flags,[release,confirm,name] fix: types apollo/client.ts

### Summary:

- **Feature**: Introduced CLI flags tailored to game release: `release`, `confirm`, and `name`.
- **Bugfix**: Addressed type issues in `apollo/client.ts` based on referenced Apollo Client issue.

### Details:

1. **CLI Flags**:
   - `release`: Marks your game as ready for release.
   - `confirm`: Confirms the intention to override existing game files.
   - `name`: Specifies the name of the game to be overridden.

2. **apollo/client.ts**:
   - Type adjustments have been made in line with the discussion on [Apollo Client Issue #6011](https://github.com/apollographql/apollo-client/issues/6011#issue-574902881).
   - Resolved potential type mismatches to prevent issues stemming from this reference.

### Testing:

1. Ensure that the `release` flag properly marks games for release.
2. Use the `confirm` flag to override files and ensure it asks for confirmation.
3. Test the `name` flag to specify different game names for overriding.
4. Validate that the type adjustments in `apollo/client.ts` align with the mentioned Apollo Client issue and that no type errors arise.

---

Thanks for reviewing. Feedback and additional insights are highly valued.
